### DISCUSSION TITLE
indent scope highlight does not work

### DISCUSSION THREAD

**Discussion by ian-ie** [folke/snacks.nvim#409] [Q&A] [answered]
![20241230112904_rec_](https://github.com/user-attachments/assets/6a62a616-0895-4bc1-8926-fc2d33a5c10b)

**Comment by Muizzyranking** [comment]
The gif seems to be working.

**Reply by ian-ie** [reply]
I set up 8 colours, however, different indents only have two colours.

**Reply by folke** [reply, marked as answer]
Yes, there's a bug currently where I select the indent hl group based on the spaces indent size.
Since you have tab=4 spaces, you will only see two different indent colors.
Will fix later

**Reply by ian-ie** [reply]
thanks！
